solidity/ecdsa/test/WalletRegistry.WalletCreation.test.ts

```diff
@@ -2365,38 +2389,6 @@ describe("WalletRegistry - Wallet Creation", async () => {
                   })
                 }
               )
-
-              context("with token staking seize call failure", async () => {
```

Comment:
TokenStaking never reverts for `seize` call assuming parameters are correct and the caller is authorized to slash. We could have this test but it would complicate the deployment and I think it is not worth the complication given this is not a real-world scenario.

Reply:
I need to think about it / play with it a bit more.

Reply:
What if I open an issue to address it separately from this PR? I tried some options and it is generally getting complicated because from one side we are trying to test with a real `TokenStaking` contract and from the other, we are trying to force it to behave in a way it is never behaving. 

I tried Smock mocks but we then need to set up half of the world that is already set up in `index.ts`. 

I have also tried altering `index.ts` but making it way more complicated for one test feels wrong. `await deployments.fixture(["WalletRegistry"])` deploys real `TokenStaking` contract and we either are not going to use it or we need to somehow parametrize it to use a stub staking contract.

I do not want to extend `TokenStaking` by some stub and use it in tests - this approach sucks because we are either testing against a real contract or not.

Reply:
Captured it in https://github.com/keep-network/keep-core/issues/2870